src/common/tracer.h

```diff
@@ -98,7 +98,7 @@ struct Tracer {
   jspan start_trace(std::string_view, bool enabled = true) { return {}; }
   jspan add_span(std::string_view, const jspan&) { return {}; }
   jspan add_span(std::string_view span_name, const jspan_context& parent_ctx) { return {}; }
-  void init(std::string_view service_name) {}
+  void init(CephContext* _cct, std::string_view service_name) {}
```

Comment:
Nit: I would suggest moving this up, before `is_enabled()` declaration, to have the stub match the real thing in `HAVE_JAEGER` branch.

Reply:
Sure, I'll fix it.